### DISCUSSION TITLE
House load calculation not showing proper

### DISCUSSION THREAD

**Discussion by CosAdrPala** [slipx06/sunsynk-power-flow-card#575] [Q&A]
Can any of you help me understand what I configured wrong that the value of the instant load is not showing properly:
![image](https://github.com/user-attachments/assets/5f3aad6c-7cce-41cf-ae63-9919a8dc6d7c)
Or how can I hide it from view since the CT from the inverter shows the complete load?

**Comment by BugsBunnyRSA** [comment]
Also the house load value is not clickable, hence cant show trend straight from here. Can that value (calculation) not be replaced with a entity of my choosing?

**Comment by slipx06** [comment]
@CosAdrPala  and @BugsBunnyRSA  Yes you can use the `essential_power` entity. If not defined the card uses internal logic to calculate this value and there is no state history so it wont be clickable

![image](https://github.com/user-attachments/assets/7dd3c72d-7943-4e88-81b8-79f39eec3c20)
